### ISSUE TITLE
isoline_demo.py fails: AttributeError: module 'os' has no attribute 'makedir'

### ISSUE THREAD

**Issue by yurivict** [jared-hughes/isosurfaces#7]
```
Traceback (most recent call last):
  File "/usr/ports/math/py-isosurfaces/files/isoline_demo.py", line 171, in <module>
    os.makedir("out")
AttributeError: module 'os' has no attribute 'makedir'
*** Error code 1
```

Version: 0.1.1
Python-3.9

**Comment by jared-hughes** [comment]
Lol, I changed `os.makedirs` to `os.makedir` instead of `os.mkdir` and didn't notice the issue because the dir was already created.

**Comment by jared-hughes** [comment]
Fixed in 43caf011d6aec0fad658f601d96d89dea606bdd5, and released in 0.1.2
